server/app/filters/NonUserRoutePrefixes.java

```diff
@@ -14,7 +14,8 @@ enum NonUserRoutePrefixes {
   DEV("/dev/"),
   FAVICON("/favicon"),
   PLAY_INDEX("/playIndex"),
-  METRICS("/metrics");
+  METRICS("/metrics"),
+  NONE("/");
```

Comment:
Added this so guest sessions aren't automatically created when we hit the home controller.